source/_page_templates/ability_template.rst

```diff
@@ -0,0 +1,58 @@
+.. [insert ability name here, lowercase, one word]:
+############
+ABILITY NAME
+############
+
+Description
+###########
+
+Paste description here. Insert gif here |ABILNAMEHERE1|
+
+
+Usage
+#####
+Below are the steps to use this ability
```

Comment:
Is this going to stay after info is replaced?

Reply:
It doesn't have to